scripts/blogView.js

```diff
@@ -0,0 +1,63 @@
+var blogView = {};
+
+blogView.populateFilter = function() {
```

Comment:
This no longer works because it's called when the DOM is ready (from the $(document).ready()), but now your stories are populated from an AJAX call. AJAX has latency, so the DOM is ready before the AJAX call returns. How can we make sure these functions are called after the AJAX call returns?